Makefile

```diff
@@ -25,7 +25,7 @@ $(TARGETS): image
 	docker run -v $(shell pwd):/app \
 		-e CARGO_HOME=/app/.cargo \
 		-e POETRY_VIRTUALENVS_PATH=/app/.virtualenvs \
-		-it $(IMAGE_NAME) \
```

Comment:
Found this broke if a target is run from CI, since it's a headless and doesn't provide a tty. Since this is used simply for building, I figured it's safe to remove. Note that the `shell` target above still retains `-it`.

Reply:
The [CI config for the C++ SDK](https://github.com/foxglove/foxglove-sdk/blob/main/.github/workflows/c_cpp.yml) is setting up a full Rust/C++ environment and doing a build directly from the checkout. I wanted to avoid pulling all that logic in here, and found that invoking `make build-cpp` (which defers all the env setup to the repo's Dockerfile) was the most minimalist approach.

Good to know about `-t` :)